tests/components/homematic/test_notify.py

```diff
@@ -1,77 +1,65 @@
 """The tests for the Homematic notification platform."""
 
-import unittest
-
 import homeassistant.components.notify as notify_comp
-from homeassistant.setup import setup_component
-
-from tests.common import assert_setup_component, get_test_home_assistant
+from homeassistant.setup import async_setup_component
 
+from tests.common import assert_setup_component
 
-class TestHomematicNotify(unittest.TestCase):
-    """Test the Homematic notifications."""
 
-    def setUp(self):  # pylint: disable=invalid-name
-        """Set up things to be run when tests are started."""
-        self.hass = get_test_home_assistant()
-        self.addCleanup(self.tear_down_cleanup)
+async def test_setup_full(hass):
+    """Test valid configuration."""
+    await async_setup_component(
+        hass,
+        "homematic",
+        {"homematic": {"hosts": {"ccu2": {"host": "127.0.0.1"}}}},
+    )
+    with assert_setup_component(1) as handle_config:
+        assert await async_setup_component(
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6333224)